packages/common/src/util/interfaces.ts

```diff
@@ -240,4 +240,5 @@ export interface IRelayConsumerParams {
   onTask?: Function
   setup?: Function
   ready?: Function
+  tearDown?: Function
```

Comment:
can be `teardown` (its technically a single word)

```suggestion
  teardown?: Function
```